src/styles/styles.js

```diff
@@ -1,151 +1,97 @@
-import v from './variables';
```

Comment:
This is where all of the shared CY-UI styles are collected. Still needs work on the typography and utilities.